repository/src/test/java/org/alfresco/repo/content/ContentServiceImplUnitTest.java

```diff
@@ -164,6 +193,17 @@ public void testRequestContentDirectUrl_StoreIsEnabledButNotImplemented()
         verify(mockContentStore, times(1)).requestContentDirectUrl(anyString(), eq(true), anyString(), anyString(), anyLong());
     }
 
+    @Test
+    public void testRequestContentDirectUrlwithQname_StoreIsEnabledButNotImplemented()
+    {
+        setupSystemWideDirectAccessConfig(ENABLED);
+        when(mockContentStore.isContentDirectUrlEnabled()).thenReturn(ENABLED);
+
+        DirectAccessUrl directAccessUrl = contentService.requestContentDirectUrl(NODE_REF, QNAME, true, 20L);
+        assertNull(directAccessUrl);
+        verify(mockContentStore, times(1)).requestContentDirectUrl(anyString(), eq(true), anyString(), anyString(), anyLong());
+    }
+
```

Comment:
Not needed. Just update the method signature in the old test. Once deprecated the old signature should eventually be removed from the product.